src/components/ShareForm.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { shareList } from '../api/firebase';
+import { auth } from '../api/config';
+
+const ShareForm = ({ listPath }) => {
+	const [email, setEmail] = useState('');
+	const [message, setMessage] = useState('');
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		if (email) {
+			try {
+				const response = await shareList(listPath, auth.currentUser.uid, email);
+
+				if (response.status === true) {
```

Comment:
I wanted to leave a quick note here since I realized it when you were sharing your screen. But since you are setting the message no matter what you can remove the if statement.